src/index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-content">
+      <section class="calendar">
+        <div class="calendar__day start-day"></div>
```

Comment:
```suggestion
      <section class="calendar calendar--start-day--wed calendar--month-length--30" >
        <div class="calendar__day"></div>
```

Please use modifiers to implement this task to be able to change start day and length